src/components/aboutModal/__tests__/__snapshots__/aboutModal.test.js.snap

```diff
@@ -68,64 +68,18 @@ Object {
 
 exports[`AboutModal Component should render a connected component with default props: connected 1`] = `
 <AboutModal
-  altLogo="ER"
-  className=""
-  logo="logo.svg"
-  onHide={[Function]}
-  productTitle={
-    <img
-      alt="product discovery"
-      src="title.svg"
-    />
-  }
+  brand={false}
+  getStatus={[Function]}
+  getUser={[Function]}
+  i18n={Object {}}
+  resetTimer={3000}
+  serverVersion="0.0.0.0000000"
   show={true}
-  trademarkText=""
->
-  <div
-    aria-label="Application information copied"
-    aria-live="polite"
-    tabIndex={-1}
-  >
-    <AboutModalVersions
-      className="quipucords-about-modal-list"
-    >
-      <AboutModalVersionItem
-        className=""
-        label="Username"
-        versionText="lorem"
-      />
-      <AboutModalVersionItem
-        className=""
-        label="Server Version"
-        versionText="0.0.0.0000000"
-      />
-      <AboutModalVersionItem
-        className=""
-        label="UI Version"
-        versionText="0.0.0.0000000"
-      />
-    </AboutModalVersions>
-  </div>
-  <div
-    className="quipucords-about-modal-copy-footer"
-  >
-    <Button
-      active={false}
-      block={false}
-      bsClass="btn"
-      bsStyle="default"
-      className="quipucords-about-modal-copy-button"
-      disabled={false}
-      onClick={[Function]}
-      title="Copy application information"
-    >
-      <Icon
-        name="paste"
-        type="fa"
-      />
-    </Button>
-  </div>
-</AboutModal>
+  t={[Function]}
+  tReady={true}
+  uiVersion="0.0.0.0000000"
+  username="lorem"
+/>
```

Comment:
why such a big change to the snapshot? I would've thought just some minor string changes.

Reply:
I'm glad you asked =) 

This is all the the translation component wrapper. Because we decided to pass the component through the "translation" piece here
- https://github.com/quipucords/quipucords-ui/pull/51/files#diff-233d6105e91463bd8457d5414ada705dR10

the translation "connected" aspect appears to not fire the same as the original connect. debated on whether to place a "test" condition check in the `index.js` file above. 

```
const connectTranslate = (mapStateToProps, mapDispatchToProps) => component =>
  connect(
    mapStateToProps,
    mapDispatchToProps
  )((process.env.REACT_APP_ENV !== 'test' && withTranslation()(component)) || component);
```

Eventually we opted for the sans-test condition, but willing to put it back in there, it would bring back expected testing behavior, thoughts?

Reply:
If you think the snapshot accurately depicts what is being set and it is testing what we want to test, then I am fine with it.

Reply:
Sure thing, after going over it again. I think it's a good call out, going to roll the `test` condition version. Not being able to expect `component.dive()` to behave as it did before in a snapshot is just hiding/masking/abstracting things, and something that could lead to confusion again if/when translation is expanded